src/components/dictionaryConcepts/components/MappingModal.jsx

```diff
@@ -10,16 +10,20 @@ import ExternalMapping from './ExternalMapping';
 export class MappingModal extends Component {
   constructor(props) {
     super(props);
-    const { to_concept_url, map_type, url } = props;
+    this.externalMapping = 'External Mapping';
```

Comment:
Are the variable names matching any JavaScript naming convention for constants?
Are they even constants?

Reply:
Let me fix that now